### DISCUSSION TITLE
Access manager for Remote Desktop.

### DISCUSSION THREAD

**Discussion by janssen-bart** [lithnet/access-manager#74] [Ideas]
Hi,

As we all know is having RDP open is an easy but insecure approach to take over computers but is widely used across internal networks by Administrators. Often in company's RDP is open from every where to any where. Access manager can maybe help in this. I strongly believe a JIT solution for RDP will put some extra layer to avoid lateral movement across devices.  

BTW. I really appreciate your contribution to the community tools. Well done!

Bart Janssen

**Comment by sylwester-majcher** [comment]
Yeah, for me it is very good ideal and it seems near to JIT. In fact in the real cases we very often gove acceess to RDP once and we forgot about that and in fact only permanent audit of the privilages can help. JIT for RDP will be fine :)

**Comment by ryannewington** [comment]
I can see the value in the idea here. While this release is focused on securing lateral movement of administrative access with JIT, the next release will add other JIT scenarios for support.

I'll add this to our backlog.
